alexa.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html>
+    <head>
+        <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+        <meta content="utf-8" http-equiv="encoding">
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="css/annotation.css">
+        <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+        <title>
+            Alexa overview of an URL 
+        </title>
+    </head>
+    <body>
+        <table width="100%" style="max-height: 200px; ">
+            <tr class="spaceUnder">
+                <td style="text-align: center;">
+                    <div id ="horizontal_line" style="margin-bottom: 5px"></div>
+                        <div class='container' style="background: white">
+                            <img class='resize_fit_center'
+                            src='images/logo.gif' />
+                        </div>
+                    <div id="horizontal_line" style="margin-top:3px"></div>
+                </td>
+            </tr>
+        </table>
+        <div>
+            <h2 style="text-align: center;"><strong>Alexa overview of Current URL</strong><br></h2>
+        </div>
+        <div class="container" id="container-whole">
+            <div class="row">
+                <div class="col-sm-12">
+                    <h4>Summary</h4> 
+                    <pre id="show_alexa_data" style="font-size: 20px">  
+                    </pre>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
+<script src="scripts/alexa.js"></script>
+<style>
```

Comment:
Why do you have all these CSS inside an HTML file? Isn't it better to put them in a CSS file and link it here? This will make the code very confusing. Please put the CSS in an existing or a new CSS file.